test/e2e/proxy_test.go

```diff
@@ -41,6 +41,8 @@ const (
 	// Provide enough time for Services to be realized or deleted.
 	// It should not be less than the minInterval of the proxy's runner (1s).
 	serviceDelay = 2 * time.Second
+
+	httpPrefix = "http://"
```

Comment:
what do you think about having a helper function to build these URLs:
```go
func getHttpURL(ip string, port string) string {
        return (&net.URL{
                Scheme: "http",
                Host: net.JoinHostPort(ip, port),
        }).String()
}
```

Reply:
That would be a graceful way to build URL!